### DISCUSSION TITLE
Translate plugin

### DISCUSSION THREAD

**Discussion by brucebailey** [MasaCMS/MasaCMS#94] [Q&A]
I will start with saying I have never used the translate plugin so this post could very well be me not setting it up properly.

1. Created two sites, default (English) and test translate site (French).
2. Install plugin on default site and assign to both sites with flags showing.
3. Create home page content on both sites and assign content nodes of translated content on each home page.
4. Put #m.dspObject(object='muratranslationstools')# in the nav
5. Reload site

I do not see anything showing where #4 is to change the language.
Also tried it directly in page content [m]m.dspObject(object='muratranslationstools')[/m] with nothing showing.

Found and tried this too (#m.dspObject(object="muratranslationsnavtools", objectparams={muratranslationstooltype='selectbox'})#) and get the following error:

500 Error
invalid component definition, can't find component [MuraTranslations.cfcs.translationManager]
Code:0
Type:expression

**Comment by guustnieuwenhuis** [comment]
Hi @brucebailey, I've created an issue in the MasaTranslations repo regarding this issue: https://github.com/MasaCMS/MasaTranslations/issues/2
